content/en/code_analysis/static_analysis/setup.md

```diff
@@ -159,7 +159,12 @@ ignore:
 
 ## Set up the GitHub integration 
 
-You must configure a GitHub App using the [GitHub integration tile][9] and set up the [source code integration][10] to see the offending code snippets as part of the Static Analysis results in the Datadog UI.
+You must configure a GitHub App using the [GitHub integration tile][9] and set up the [source code integration][10] to see the offending code snippets as part of the Static Analysis results in the Datadog UI. 
+
+When installing the GitHub App, the following permissions are required to enable certain features:
+
+1. `Content: Read` which will allow you to see code snippets displayed in the Datadog UI
+2. `Pull Request: Read & Write` which will allow Datadog to add feedback for violations directly in your pull requests using [PR Comments][11]
```

Comment:
```suggestion
- `Content: Read`, which allows you to see code snippets displayed in Datadog
- `Pull Request: Read & Write`, which allows Datadog to add feedback for violations directly in your pull requests using [pull request comments][11]
```

- Changed to an unordered list, as the items in this list have no inherent order or hierarchy. Ordered lists are best used when the list items need to be in a specific order, such as when describing the steps in a procedure.
- Changed future tense to present tense for styleguide
- "PR Comments" -> "pull request comments", as it is not a proper noun